### PR TITLE
codeml: fix test parameters

### PR DESCRIPTION
**Pull request by bernt-matthias**
and make all outputs valid cheetah placeholders

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/codeml/codeml.xml [hunk 1]
```diff
@@ -238,39 +238,39 @@ codeml '$codeml_ctl'
...
     <tests>
-        <test>
+        <test expect_num_outputs="9">
             <conditional name="compat_model" >
                 <param name="brmodel" value="0" />
                 <param name="NSsites" value="0" />
             </conditional>
-            <param name="adv.fix_omega" value="0" />
-            <param name="adv.omega" value="0.2" />
+            <param name="adv|fix_omega" value="0" />
```
Comment (bgruening) [inline tools/codeml/codeml.xml:266]: should we use the `<conditional>` syntax here?

File: tools/codeml/codeml.xml [hunk 2]
```diff
@@ -238,39 +238,39 @@ codeml '$codeml_ctl'
...
         <data format="txt" name="run" from_work_dir="run_codeml" label="${tool.name} on ${on_string}: run_codeml"/>
-        <data format="txt" name="4fold" from_work_dir="4fold.nuc" label="${tool.name} on ${on_string}: 4fold.nuc">
+        <data format="txt" name="fourfold" from_work_dir="4fold.nuc" label="${tool.name} on ${on_string}: 4fold.nuc">
             <filter>adv['verbose']=="1"</filter>
         </data>
     </outputs>
 
     <tests>
-        <test>
+        <test expect_num_outputs="9">
```
Comment (bgruening) [inline tools/codeml/codeml.xml:261]: @bernt-matthias should we make this also required for tests during linting?

File: tools/codeml/codeml.xml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<tool name="codeML" id="codeml" version="@TOOL_VERSION@+galaxy1" profile="18.01">
+<tool name="codeML" id="codeml" version="@TOOL_VERSION@+galaxy0" profile="18.01">
```
Comment (bgruening) [inline tools/codeml/codeml.xml:1]: @bernt-matthias why decreasing to 0?

File: tools/codeml/codeml.xml (hunk 2 above)
Comment (bernt-matthias) [reply tools/codeml/codeml.xml:261]: Already on the way https://github.com/galaxyproject/galaxy/pull/12975 :)

File: tools/codeml/codeml.xml (hunk 3 above)
Comment (bernt-matthias) [reply tools/codeml/codeml.xml:1]:
```suggestion
<tool name="codeML" id="codeml" version="@TOOL_VERSION@+galaxy2" profile="18.01">
```

File: tools/codeml/codeml.xml (hunk 1 above)
Comment (bernt-matthias) [reply tools/codeml/codeml.xml:266]: Don't care so much .. and feel really lazy :)

**Review by bgruening — APPROVED** [review]

**Pull request merged**